frontend/src/pages/NewRunV2.tsx

```diff
@@ -134,6 +134,22 @@ function getPipelineDetailsUrl(
   return isRecurring ? pipelineDetailsUrlfromRecurringRun : pipelineDetailsUrlfromRun;
 }
 
+export async function getLatestVersion(pipelineId: string) {
+  try {
+    const listVersionsResponse = await Apis.pipelineServiceApiV2.listPipelineVersions(
+      pipelineId,
+      undefined,
+      1, // Only need the latest one
+      'created_at desc',
+    );
+    return listVersionsResponse.pipeline_versions
+      ? listVersionsResponse.pipeline_versions[0]
+      : undefined;
+  } catch (err) {
+    return;
```

Comment:
Should we properly handle the error?

Reply:
I think we can either return or pop some error message. Let me add a logger for handling this error.